src/super_gradients/recipes/imagenet_resnet50_kd.yaml

```diff
@@ -25,11 +25,11 @@ val_dataloader: imagenet_val
 resume: False
 training_hyperparams:
   resume: ${resume}
-  loss: KDLogitsLoss
-  criterion_params:
-    distillation_loss_coeff: 0.8
-    task_loss_fn:
-      _target_: super_gradients.training.losses.label_smoothing_cross_entropy_loss.LabelSmoothingCrossEntropyLoss
+  loss:
+    KDLogitsLoss:
+      distillation_loss_coeff: 0.8
+      task_loss_fn:
+        _target_: super_gradients.training.losses.label_smoothing_cross_entropy_loss.LabelSmoothingCrossEntropyLoss
```

Comment:
Maybe we can also instantiate loss for `task_loss_fn` using LossFactory:

```
loss:
    KDLogitsLoss:
      distillation_loss_coeff: 0.8
      task_loss_fn: LabelSmoothingCrossEntropyLoss

```

It is not directly related to this PR, but caught my attention that it violates our instantiation paradigm

Reply:
fixed,